### PR TITLE
[13562-edit_line_style_text] edit inspector text to better describe line style

### PR DESCRIPTION
**Pull request by EricManiez**
Resolves: #13562 

In the inspector, the title for line styles was too vague, and was made more precise.

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [x] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [x] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

File: src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineStyleSection.qml [hunk 1]
```diff
@@ -70,7 +70,7 @@ Column {
 
     FlatRadioButtonGroupPropertyView {
         id: styleSection
-        titleText: qsTrc("inspector", "Style")
+        titleText: qsTrc("inspector", "Line Style")
```
Comment (cbjeukendrup) [inline src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineStyleSection.qml:73]: It's part of our design style that we don't use "Title Case" (i.e. capitalising every word). Instead, we prefer "Sentence case" everywhere.
```suggestion
        titleText: qsTrc("inspector", "Line style")
```

**Review by cbjeukendrup — DISMISSED** [review]
Thanks!

**Comment by cbjeukendrup** [comment]
By the why, the failing "VTests" check should be fixed after rebasing your branch on top of the latest master branch (usually `git pull --rebase upstream master` should do the job)

**Comment by BrownianNotion** [comment]
@cbjeukendrup Happy to resubmit this PR and resolve the issue if you want to close this one.

**Comment by cbjeukendrup** [comment]
@BrownianNotion Thanks, that would be great!

Closing due to inactivity.

**Pull request closed without merging**